.github/workflows/release_notes_check.yml

```diff
@@ -0,0 +1,26 @@
+name: Release Notes Check
+
+on:
+  pull_request:
+    types:
+      # On by default if you specify no types.
+      - "opened"
+      - "reopened"
+      - "synchronize"
+      # For `skip-label` only.
+      - "labeled"
+      - "unlabeled"
+
+jobs:
+  check-release-notes:
+    name: Check release notes are updated
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for a release notes update
+        uses: frequenz-floss/check-for-changed-files@token-support
```

Comment:
Since this is a public repo I think it is safe to use the upstream repo, and we don't need the token support either, so if you update to use upstream we can merge already (maybe use a hash instead of the version for extra security).